runtime/oti/j9consts.h

```diff
@@ -458,6 +458,8 @@ extern "C" {
 
 #define J9_GC_MINIMUM_OBJECT_SIZE 0x10
 #if defined(J9VM_ENV_DATA64)
+#define J9_GC_INDEXABLE_DATA_FIELD_SIZE sizeof(U_64)
+#define J9_GC_MINIMUM_INDEXABLE_OBJECT_SIZE J9_GC_MINIMUM_OBJECT_SIZE +  J9_GC_INDEXABLE_DATA_FIELD_SIZE
```

Comment:
Brackets please.

Reply:
nit: please also lose the extra space after `+`